nncf/quantization/algorithms/hyperparameter_tuner/param_grid.py

```diff
@@ -79,7 +80,12 @@ def _get_minmax_quantization_param_grid() -> ParamGrid:
 
 
 def _get_smooth_quant_param_grid() -> ParamGrid:
-    return {"advanced_parameters:smooth_quant_alpha": [0.15, 0.25, 0.5, 0.75, 0.95]}
+    alpha_values = [0.15, 0.25, 0.5, 0.75, 0.95]
```

Comment:
Are you planning to add alpha for convolutions later?

Reply:
Yes. I think that it's better to do some experiments with Convolution's alpha and AAQ as a separate activitiy.